src/Util/DigitizeUtil.ts

```diff
@@ -94,7 +94,10 @@ export class DigitizeUtil {
    *
    * @param map
    */
-  static getDigitizeLayer(map: OlMap): OlVectorLayer<OlFeature> {
+  static getDigitizeLayer(map: OlMap | null): OlVectorLayer<OlFeature> | null {
+    if (!map) {
+      return null;
+    }
```

Comment:
This is a breaking change i think. I was thinking of some conditional typing that could would return null if null is passed in, but that does not work. I general I think it is better to check the null before and not allow null as an input value.

Reply:
Well, that breaks the [rules of hooks](https://react.dev/reference/rules/rules-of-hooks), because if checked before and outside of the hook, the hook may or may not be called during a render. If you're picky, it's a breaking change, on the other hand the current code is already broken...

Reply:
IMHO this is definitely a breaking change.

Reply:
Alright, I've made this a breaking change.